Mail Courier Outfit Additions

RetroBit23 opened the pull request:
# Description

Adds the "Express Space Courier" set to the courierdrobe for couriers to use and a warm worn winter coat (made by @ItsDzordz ).

---

# TODO

- [x] Add Express Courier Outfit
- [x] Add Express Courier Trunk
- [x] Add Express Courier Outfit & Trunk to the CourierDrobe
- [x] Add Warm Worn Out Coat to winterdrobe

---

<details><summary><h1>Media</h1></summary>
<p>

![front](https://github.com/user-attachments/assets/7d78fd89-ca9e-4a15-820a-2bc62eea980b)
![left](https://github.com/user-attachments/assets/7898fe30-8a0c-428f-ada5-cc0683ddd5b7)
![back](https://github.com/user-attachments/assets/9c0a057e-c9f1-456e-adb7-bdb9a3edf393)
![right](https://github.com/user-attachments/assets/acd4d7e0-e111-42ab-b59f-6b6330ceaf24)

</p>
</details>

---
# Changelog

:cl: RetroBit23, ItsDzordz
add: The 'Express Courier' outfit and trunk has been added to the CourierDrobe.
add: The 'Warm wornout coat' has been added to the WinterDrobe.

RetroBit23 committed:
Adds Express Courier Outfit

RetroBit23 committed:
Update duffels.yml

RetroBit23 committed:
Small changes

RetroBit23 committed:
Trunk Size Buff and Speed Tweak

Made the trunk a bit bigger to what i wanted it to be and makes the speed reduction slightly better.

RetroBit23 committed:
Express Coat changes & Trunk Damage changes

- Made Express Courier Coat winter Insulated.

- Added throwing damage to Trunk

RetroBit23 committed:
Small correction

- Fixed the spelling error in the description.

M3739 commented on Resources/Prototypes/_Floof/Entities/Clothing/Back/duffels.yml:
```diff
@@ -0,0 +1,27 @@
+  parent: ClothingBackpack
+  id: ClothingBackpackDuffelCourierDevy
+  name: Express Courier Trunk
+  description: A Massive storage trunk for express space couriers. It has a large inside at the cost of being heavy.
```
This could be rewritten to sound more correct, especially at the "large inside" part. Here is a suggestion. Note that you can rewrite this suggestion however you like, and it's intended to give you an idea.
```suggestion
  description: A massive storage trunk issued to express space couriers. It boasts a spacious interior but is considerably heavy.
```

M3739 commented on Resources/Prototypes/_Floof/Entities/Clothing/Back/duffels.yml:
```diff
@@ -0,0 +1,27 @@
+  - type: Storage
+    maxItemSize: Huge
+    grid:
+    - 0,0,12,5
```
Resulting grid is 13 x 6. Total space is 78.
![image](https://github.com/user-attachments/assets/d221fc64-416d-4b8b-9a6e-bc0cca844840)
This sports almost double the storage capacity of a duffel bag (8 x 5 grid, total of 40), but is still less than a satchel of holding (20 x 10 grid, total of 200).

M3739 commented on Resources/Prototypes/_Floof/Entities/Clothing/Back/duffels.yml:
```diff
@@ -0,0 +1,27 @@
+    - 0,0,12,5
+  - type: ClothingSpeedModifier
+    walkModifier: 0.9
+    sprintModifier: 0.75
```
25% sprint penalty, and 10% walk penalty. Hefty. Comparable to wearing a juggernaut suit, which has a 35% sprint penalty and 10% walk penalty. 

Since we are essentially carrying what is a whole large hard-shell luggage with some serious capacity, I think this is fine.

M3739 commented on Resources/Prototypes/_Floof/Entities/Clothing/Back/duffels.yml:
```diff
@@ -0,0 +1,27 @@
+      bluntStaminaDamageFactor: 2
+    heavyStaminaCost: 15
+    soundHit:
+      collection: MetalThud
```
I am unsure about this. I'll have to discuss with my peers.

Raikyr0 replied:
The vision I do believe is so it also morphs into a weapon, like it does in game.

M3739 replied:
After some discussion a while back, we're going with this as is.

I will say, good job on the balancing.

M3739 commented on Resources/Prototypes/_Floof/Entities/Clothing/departmental_clothes.yml:
```diff
@@ -459,6 +459,39 @@
+  parent: ClothingUniformBase
+  id: ClothingUniformCourierDevy
+  name: Express Courier Lining
+  description: a white lining for express space couriers.
```
```suggestion
  description: A white lining for express space couriers.
```

M3739 reviewed (changes requested):
Project moon mentioned.

Anyways, only thing I will request for you to change is the description of the trunk. The rest is something I have to discuss with others, and so it's to give some reference by translating into a more clear picture for both you and others who may review this PR.

In addition, your changelog has been rewritten to be more concise. Careful not to add too many lines in it, and feel free to adjust it accordingly.

Memeji reviewed (approved):
Like the idea, gives Courier something cool.
Just need to modify the descriptions as M3739 has shown and make sure the tests pass!

M3739 commented:
Also,  forgot to mention, YAML Linter test fail is valid:
```
Error: /Prototypes/_Floof/Entities/Clothing/Back/duffels.yml(22,7)  Field "bluntStaminaDamageFactor" not found in "Content.Shared.Damage.DamageSpecifier".
```

RetroBit23 committed:
Name & Description Changes + Trunk Tweak

Changed and Polished Names and Descriptions.
Removed Trunk damage overall, wasn't very happy with it.
